### PR TITLE
Render road ref along them, smaller cycle route shield, cleaning

### PR DESCRIPTION
**Pull request by Florimondable**
link to #197 

cyclosm_road_route_ref_city
cyclosm_road_names_country

### CODE REVIEW

File: labels.mss [hunk 1]
```diff
@@ -129,166 +129,103 @@
...
+      text-clip: false;
+      text-placement: line;
+      text-face-name: @sans;
+      text-repeat-distance: @major-highway-text-repeat-distance;
+      text-halo-radius: @standard-halo-radius;
   }
 }
 
 #roads-text-ref {
-  [highway = 'motorway_trunk'],
```
Comment (Phyks) [inline labels.mss]: The two of them were merged https://github.com/cyclosm/cyclosm-cartocss-style/blob/master/project.mml#L2011 in SQL queries. We should either keep `motorway_trunk` or drop the merge in `roads-text-ref` SQL.

File: labels.mss [hunk 2]
```diff
@@ -129,166 +129,103 @@
...
+      text-size: 12;
     }
   }
 }
 
 #roads-text-name {
   [highway = 'motorway'],
   [highway = 'trunk'],
-  [highway = 'primary'],
-  [highway = 'construction'][construction = 'motorway'],
```
Comment (Phyks) [inline labels.mss]: Why dropping the name of roads under construction?

File: labels.mss [hunk 3]
```diff
@@ -629,6 +521,7 @@
   shield-face-name: @sans;
   shield-clip: @shield-clip;
   shield-file: url("symbols/shields/[type]_[width]x[height].svg");
+  shield-transform: "scale(0.75)";
```
Comment (Phyks) [inline labels.mss]: I'd rather edit the script https://github.com/cyclosm/cyclosm-cartocss-style/blob/master/scripts/generate_shields.py to generate shield the right size. I'll handle this after this PR has been merged.

File: project.mml [hunk 4]
```diff
@@ -2101,20 +2100,16 @@ Layer:
...
           ('tertiary_link', 200),
-          ('service', 150),
-          ('cycleway', 140),
-          ('path', 130),
-          ('construction', 10)
+          ('service', 150)
         ) AS ordertable (highway, prio)
         USING (highway)
         WHERE highway IN ('motorway', 'motorway_link', 'trunk', 'trunk_link', 'primary', 'primary_link', 'secondary', 'secondary_link', 'tertiary',
-          'tertiary_link', 'residential', 'unclassified', 'road', 'service', 'pedestrian', 'raceway', 'living_street', 'construction', 'cycleway', 'path')
```
Comment (Phyks) [inline project.mml]: With these edits, we lost the names of raceways, is it expected?

**Comment by Phyks** [comment]
:+1:

We have a bit of readability issues at low zooms (city or less).

https://cyclosm.org/#13/48.8779/2.2867

Screenshot_2019-11-08 CyclOSM — Kosmtik

https://cyclosm.org/#11/48.8401/2.0098

Screenshot_2019-11-08 CyclOSM — Kosmtik 2

File: labels.mss (hunk 2 above)
Comment (Florimondable) [reply labels.mss]: What’s the point of having them ?

File: labels.mss (hunk 3 above)
Comment (Florimondable) [reply labels.mss]: I tried, I failed 😭

File: project.mml (hunk 4 above)
Comment (Florimondable) [reply project.mml]: Yes, I don’t see the point of having raceway names in a pov of a cyclist.

**Comment by Florimondable** [comment]
> We have a bit of readability issues at low zooms (city or less).
May be, not sure, +1 to the font size ?

File: labels.mss (hunk 1 above)
Comment (Florimondable) [reply labels.mss]: I don’t remind to have done that change, must come from the copy past I have done to write that layer from name layer.

File: labels.mss (hunk 2 above)
Comment (Phyks) [reply labels.mss]: We'd lose the name on "Avenue de la République" here for instance https://www.cyclosm.org/#map=19/48.80997/2.30184/cyclosm.

Comment (Florimondable) [reply labels.mss]: The road is not even rendered, the name is on nothing, so it is a good fix :]

**Pull request merged**